config/AMBASSADORS_MEMBERS.json

```diff
@@ -1,164 +1,176 @@
-			},
-			{
-				"type": "presentation",
-				"title": "Automated Apache Kafka Mocking and Testing with AsyncAPI",
-				"date": {
-					"year": 2021,
-					"month": "September"
-				},
-				"link": "https://www.confluent.io/events/kafka-summit-americas-2021/automated-apache-kafka-mocking-and-testing-with-asyncapi/"
-			},
-			{
-				"type": "presentation",
-				"title": "Event-driven APIs & Schema governance for Apache Kafka, Apidays LIVE Hong Kong",
-				"date": {
-					"year": 2021,
-					"month": "August"
-				},
-				"link": "https://www.youtube.com/watch?v=rwlHJkgaRdw"
-			},
-			{
-				"type": "presentation",
-				"title": "Event-driven APIs & Schema governance for Apache Kafka, API Specification Conference",
-				"date": {
-					"year": 2021,
-					"month": "August"
-				},
-				"link": "https://www.youtube.com/watch?v=QZ4cJyTw-U0"
-			},
-			{
-				"type": "presentation",
-				"title": "Getting started with Event-Driven APIs, Apidays LIVE Paris",
-				"date": {
-					"year": 2021,
-					"month": "December"
-				},
-				"link": "https://www.youtube.com/watch?v=_kIE-qtpwTA&t=9s"
-			}
-		]
-	},
-	{
-		"name": "Jesse Menning",
-		"github": "jessemenning",
-		"twitter": "JesseMenning",
-		"linkedin": "jesse-menning",
-		"company": "Solace",
-		"contributions": [
-			{
-				"type": "article",
-				"title": "AsyncAPI 2.3 Adds Solace Bindings to Unlock Full Power of PubSub+",
-				"date": {
-					"year": 2022,
-					"month": "January"
-				},
-				"link": "https://solace.com/blog/asyncapi-adds-solace-bindings/"
-			},
-			{
-				"type": "article",
-				"title": "Align Production Reality and Event Documentation with the AsyncAPI Discovery Tool",
-				"date": {
-					"year": 2021,
-					"month": "November"
-				},
-				"link": "https://solace.com/blog/asyncapi-discovery-tool/"
-			},
-			{
-				"type": "article",
-				"title": "AsyncAPI vs. OpenAPI: Answers to Your Burning Questions About the Two Leading API Specs",
-				"date": {
-					"year": 2021,
-					"month": "June"
-				},
-				"link": "https://solace.com/blog/asyncapi-vs-openapi/"
-			},
-			{
-				"type": "article",
-				"title": "AsyncAPI, CloudEvents, OpenTelemetry: Which Event-Driven Specs Should Your DevOps Include?",
-				"date": {
-					"year": 2021,
-					"month": "March"
-				},
-				"link": "https://solace.com/blog/asyncapi-cloudevents-opentelemetry-event-driven-specs-devops/"
-			},
-			{
-				"type": "presentation",
-				"title": "Future of AsyncAPI, Panel discussion, AsyncAPI Conference",
-				"date": {
-					"year": 2021,
-					"month": "November"
-				},
-				"link": "https://www.youtube.com/watch?v=3EeMHhbwyOQ"
-			},
-			{
-				"type": "presentation",
-				"title": "Thinking Out Loud #4 — with Jesse Menning",
-				"date": {
-					"year": 2021,
-					"month": "August"
-				},
-				"link": "https://www.youtube.com/watch?v=eJewFUR-OaM"
-			}
-		]
-	}
+  {
+    "name": "Alejandra Quetzalli",
+    "img": "/img/avatars/canela-ale.webp",
```

Comment:
we should not go this direction. This will complicate life. Ambassador will not only have to be added to `community` repo but also `website`. Please remove and like in case of TSC members list, take the photo of ambassador from GitHub profile